model/riscv_pmp_control.sail

```diff
@@ -80,14 +79,21 @@ function pmpAddrRange(cfg: Pmpcfg_ent, pmpaddr: xlenbits, prev_pmpaddr: xlenbits
   match pmpAddrMatchType_of_bits(cfg.A()) {
     OFF   => None(),
     TOR   => { Some ((prev_pmpaddr << 2, pmpaddr << 2)) },
-    NA4   => { // TODO: I find the spec unclear for entries marked NA4 and G = 1.
-               // (for G >= 2, it is the same as NAPOT). In particular, it affects
-               // whether pmpaddr[0] is always read as 0.
+    NA4   => {
+               // NA4 is not selectable when the PMP grain G >= 1. See pmpWriteCfg().
+               assert(sys_pmp_grain() >= 1, "NA4 cannot be selected when PMP grain G >= 1.");
```

Comment:
`When G ≥ 1, the NA4 mode is not selectable`
```suggestion
               assert(sys_pmp_grain() < 1, "NA4 cannot be selected when PMP grain G >= 1.");
```
This check seems inverted?

Reply:
Ooops good spot!